tensorflow/tools/docker/Dockerfile

```diff
@@ -23,10 +23,11 @@ RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
     python get-pip.py && \
     rm get-pip.py
 
+RUN python2 -m pip install ipykernel
```

Comment:
Can you help me understand why these changes related to ipykernel install is needed?

Reply:
I turned to this page http://ipython.readthedocs.io/en/stable/install/kernel_install.html, and when I build using the org. one, the errors appeared, and the image can not be built correctly.

Reply:
We have nightly builds on TensorFlow Jenkins that are based on this file. They have been consistently passing in recent days. Please see the log at:
http://ci.tensorflow.org/view/Nightly/job/nightly-docker-cpu/

The console logs contain the commands used in the builds.